modules/contact/src/postprocessors/MortarContactAreaPostprocessor.C

```diff
@@ -0,0 +1,45 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MortarContactAreaPostprocessor.h"
+
+registerMooseObject("ContactApp", MortarContactAreaPostprocessor);
+
+InputParameters
+MortarContactAreaPostprocessor::validParams()
+{
+  InputParameters params = VolumePostprocessor::validParams();
+
+  params.addClassDescription(
+      "Computes the surface area in contact by comparing summing over all lower dimensional blocks "
+      "on contact area that have a non-zero Lagrange Multiplier");
+  params.addRequiredCoupledVar("variable", "Coupled Lagrange Multiplier");
+  params.addParam<Real>(
+      "threshold",
+      1.0,
```

Comment:
Why 1.0 as the default? Wouldn't 0.0 be a safer (unit independent) choice?

But IMO anything that uses a threshold in the first place is iffy.

Reply:
In this test (and elsewhere) the LM doesn't always return to 0 when not in contact. 1.0 was a simple number here, since that means 1.0 a bit of contact occurs. Perhaps when units are in MPa, 1.0 MPa is too high, so maybe 1.0e-6 or something like that is better? It's still a magic number, I get it, but something is needed here otherwise once in contact, "always in contact" for this method.

Reply:
Should probably be a fuzzyGreaterThan and the fuzz should be of the order of the solver tolerance.

Reply:
I compared to tolerance, following similar methods in `ContactDOFSetSize`